examples/k8s.yaml

```diff
@@ -54,10 +54,46 @@ provision:
     apt-get install -y cri-tools
     cat  <<EOF | sudo tee /etc/crictl.yaml
     runtime-endpoint: unix:///run/containerd/containerd.sock
-    image-endpoint: unix:///run/containerd/containerd.sock
     EOF
     # cni-plugins
     apt-get install -y kubernetes-cni
+    mkdir -p /etc/cni/net.d
+    cat << EOF | tee /etc/cni/net.d/10-containerd-net.conflist
+    {
+      "cniVersion": "0.4.0",
+      "name": "containerd-net",
+      "plugins": [
+        {
+          "type": "bridge",
+          "bridge": "cni0",
+          "isGateway": true,
+          "ipMasq": true,
+          "promiscMode": true,
+          "ipam": {
+            "type": "host-local",
+            "ranges": [
+              [{
+                "subnet": "10.88.0.0/16"
+              }],
+              [{
+                "subnet": "2001:4860:4860::/64"
+              }]
+            ],
+            "routes": [
+              { "dst": "0.0.0.0/0" },
+              { "dst": "::/0" }
+            ]
+          }
+        },
+        {
+          "type": "portmap",
+          "capabilities": {"portMappings": true}
+        }
+      ]
+    }
+    EOF
+    # To use flannel, delete the default CNI network
+    # To use containerd-net, comment this and below
     rm -f /etc/cni/net.d/*.conf*
```

Comment:
This rm -f is confusing

Reply:
I think I inherited from the installation where we also have podman and cri-o

Perhaps remove the glob and use the above only ? Hope nothing else is there.
Kubernetes only supports having **one** file in CNI, and uses the first in alphabet.

Reply:
In the recent release, the cni conf dir flag has been removed - which broke our workaround

Reply:
Anyway, we get a default CNI with the container runtime installation.

It must be deleted, if we want to install our own CNI - like "flannel"

Reply:
I think we should only support flannel.
Not really useful for single-node cluster, but akin to production env.

Reply:
Main reason for containerd-net was to not depend on any third-party images

Reply:
The registry changed from k8s.gcr.io to registry.k8s.io and from quay.io to docker.io, but anyway.

```
IMAGE                                     TAG                 IMAGE ID            SIZE
docker.io/flannel/flannel-cni-plugin      v1.1.2              7a2dcab94698c       3.84MB
docker.io/flannel/flannel                 v0.21.2             7b7f3acab868d       24.2MB
registry.k8s.io/coredns/coredns           v1.9.3              5185b96f0becf       14.8MB
registry.k8s.io/etcd                      3.5.6-0             fce326961ae2d       103MB
registry.k8s.io/kube-apiserver            v1.26.1             deb04688c4a35       35.3MB
registry.k8s.io/kube-controller-manager   v1.26.1             e9c08e11b07f6       32.2MB
registry.k8s.io/kube-proxy                v1.26.1             46a6bb3c77ce0       21.5MB
registry.k8s.io/kube-scheduler            v1.26.1             655493523f607       17.5MB
registry.k8s.io/pause                     3.9                 e6f1816883972       322kB

```